### PR TITLE
add license of jquery.cycle,cc #5500

### PR DESCRIPTION
**Pull request by dada0423**
https://github.com/malsup/cycle/blob/master/cycle.jquery.json#L16
@pvnr0082t

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@dada0423 please confirm that your git client been configured with correct user info as GitHub can't link the author to your GitHub account.

File: ajax/libs/jquery.cycle/package.json [hunk 1]
```diff
@@ -24,5 +24,6 @@
         ]
       }
     ]
-  }
+  },
+"license" :"MIT"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.cycle/package.json]: Wong indent.

File: ajax/libs/jquery.cycle/package.json [hunk 2]
```diff
@@ -24,5 +24,9 @@
         ]
       }
     ]
-  }
+  },
+  "licenses":[
```
Comment (pvnr0082t) [inline ajax/libs/jquery.cycle/package.json]: @dada0423 It need a space after colon mark.

**Comment by pvnr0082t** [comment]
@dada0423 Any update?

**Comment by pvnr0082t** [comment]
@dada0423 You should ping me after you fixed the problem.

**Comment by pvnr0082t** [comment]
LGTM
@cdnjs/intern2 Please help review this PR, thanks.

**Comment by x09326** [comment]
LGTM

**Comment by pvnr0082t** [comment]
ping @PeterDaveHello

**Pull request merged**